library/src/main/java/com/bumptech/glide/GlideBuilder.java

```diff
@@ -245,6 +247,18 @@ public GlideBuilder setConnectivityMonitorFactory(ConnectivityMonitorFactory fac
   }
 
   /**
+   * Sets the {@link
+   * UncaughtThrowableStrategy} to use to
+   *                                  handle uncaught exceptions.
+   *
+   * @param uncaughtThrowableStrategy The strategy to use.
+   * @return This builder.
+   */
+  public void setUncaughtThrowableStrategy(UncaughtThrowableStrategy uncaughtThrowableStrategy) {
```

Comment:
Since you can already set the disk cache and source executors and I expect this to be relatively rare, you can probably remove these methods on GlideBuilder and let users pass in their own GlideExecutors instead if they want to set the UncaughtThroawableStrategy.